bower.json

```diff
@@ -12,6 +12,7 @@
     "fontawesome": "~4.3.0",
     "highcharts": "~4.1.5",
     "jquery": "^1.11.1",
+    "jquery-mousewheel": "^3.1.12",
```

Comment:
If https://github.com/softlayer/sl-ember-components/pull/434#discussion_r35028305 resolved bug https://github.com/softlayer/sl-ember-components/issues/433 then how is this change related or why is it needed?  Is it adding anything to the fix?  If it is something that has been determined as being desired for the functionality of this component then it should be done in a separate issue.